include/aws/crt/io/Bootstrap.h

```diff
@@ -44,11 +57,36 @@ namespace Aws
                 operator bool() const noexcept;
                 int LastError() const noexcept;
 
+                /**
+                 * Set function to invoke when ClientBootstrap's behind-the-scenes
+                 * resources finish shutting down. This function may be invoked
+                 * on any thread. Shutdown begins when the ClientBootstrap's
+                 * destructor runs.
+                 */
+                void SetShutdownCompleteCallback(OnClientBootstrapShutdownComplete callback);
```

Comment:
why not noexcept? move asignment is supposed to be noexcept by definition right?

Reply:
nope, `function`'s [assignment operators](https://en.cppreference.com/w/cpp/utility/functional/function/operator%3D) are mostly excepty